### DISCUSSION TITLE
Hide "Powered by Jekyll & Minimal Mistakes." in footer

### DISCUSSION THREAD

**Discussion by ghost** [mmistakes/minimal-mistakes#3193] [Q&A] [answered]
How can I hide the message "Powered by Jekyll & Minimal Mistakes." in the footer?

**Comment by iBug** [comment, marked as answer]
Override [`_includes/footer.html`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/footer.html)

**Reply by ghost** [reply]
Should I make the changes in [`_includes/footer/custom.html`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/footer/custom.html)

**Reply by iBug** [reply]
No, you need to [override that file](https://mmistakes.github.io/minimal-mistakes/docs/overriding-theme-defaults/).

**Reply by ghost** [reply]
Thanks
